### DISCUSSION TITLE
Gitignore

### DISCUSSION THREAD

**Discussion by Ultra-T** [community/community#72135] [Actions] [answered]
### Select Topic Area

Question

### Body

Greetings, i would kindly like to ask if there exists a gitignore which can be utilized for UiPath and PowerBI

**Comment by andrej-kocijan** [comment, marked as answer]
Can't seem to find an universally used one myself but maybe this helps:
```.gitignore
# UiPath specific
project.json
*.nupkg
*.nupkg.sha512
*.xaml
*.xaml.cs
*.log
*.bak
*.tmp
*.xlsx

# Power BI specific
*.pbix
*.pbit
```

**Comment by ShonHarsh** [comment]
Hello @Ultra-T  @andrej-kocijan ,
The previous post that includes the .xaml files is incorrect.

Regarding the .xaml files, if you ignore these then your workflows will not be selected for committing.  My understanding is that a developer would want to include the files they are working on for source control.  These .xaml files would be similar to a .cs file for C# or a .py file for Python.

This is my .gitignore file for UiPath:
```

##################################
### UiPath RPA Development      ###
##################################
**/*.log
**/.local/**
**/.objects/**
**/.settings/**
**/.tmh/**
